Localization/de-DE/start-folding.yaml

```
@@ -0,0 +1,47 @@
+---
+url: "https://foldingathome.org/"
+language: "de-de"
+title: "BEGINNE ZU FALTEN"
+text: [
+# Bold first paragraph
+#"The Folding@home software runs while you do other things.",
+"Die Folding@home Software läuft im Hintergrund.",
+
+
+# Second paragraph
+#"While you keep going with your everyday activities, your computer will be working to help us find cures for diseases like cancer, ALS, Parkinson’s, Huntington’s, Influenza and many others.",
+"Während Sie mit Ihren alltäglichen Dingen weitermachen, wird Ihr Computer uns dabei helfen Heilmittel für Krankheiten wie Krebs, ALS, Parkinson, Huntington, Grippe und viele andere zu finden.",
+
+# Third paragraph first sentence in bold
+#"Please be patient if you experience idle time as we face the COVID-19 pandemic together! ",
+"Bitte haben Sie etwas Geduld, falls Ihr Computer zwischendurch nichts zu tun hat, da wir alle von der COVID-19 Pandemie betroffen sind! ",
+
+# Rest of the third paragraph
+#"We are working like mad to start simulations of COVID-19 proteins. The enthusiastic response to our effort has been tremendous and has sometimes emptied our queues. This is a great “problem” to have as there is no end to the valuable science we can do, its just a matter of giving us some time to get more running.",
+"Wir arbeiten mit Hochdruck daran um mit Simulationen von COVID-19 Proteinen anzufangen. Die enthusiastischen Reaktionen auf unsere Bemühungen waren enorm und dies hat die Warteschlangen manchmal geleert. Das ist ein großartiges "Problem", denn die wertvollen Wissenschaft zu der wir fähig sind ist nicht abgeschlossen. Es geht bloß darum, uns etwas mehr Zeit zu geben, um mehr in Gang zu bringen.",
+
+#
+# OS-dependent binary download box
+# TODO: needs translation as well!
+
+# Bold text
+#"Don’t see the correct software for your device? Try our list of",
+"Sie finden nicht die richtige Software für Ihr Gerät? Probieren Sie unsere Liste mit ",
```

```suggestion
"Sie finden nicht die richtige Software für Ihr Gerät? Probieren Sie unsere Liste mit",
```

Not sure how important additional spaces are.

---

The additional space is because the sentence is not finished there, it continuous on the next part. In the English original it is the same.

---

The English original doesn't have a space there. That's why I pointed it out. :) Still not sure how important it is and it might not even matter at all as HTML ignores multiple spaces.